chapters/hpmor-chapter-000.tex

```diff
@@ -15,27 +15,26 @@ \chapter*{Author’s introduction}
-\item DUMBLEDORE is either insane, or playing some vastly deeper game which involved setting fire to a chicken.
-\item DEPUTY HEADMISTRESS MINERVA MCGONAGALL needs to go off somewhere private and scream for a while.
+\item \textsc{Petunia Evans} married Michael Verres, a Professor of Biochemistry at Oxford.
+\item \textsc{Harry James Potter-Evans-Verres} grew up in a house filled to the brim with books. He once bit a math teacher who didn't know what a logarithm was. He's read \emph{Gödel, Escher, Bach} and \emph{Judgment Under Uncertainty: Heuristics and Biases} and volume one of \emph{The Feynman Lectures on Physics}. And despite what everyone who's met him seems to fear, he doesn't want to become the next Dark Lord. He was raised better than that. He wants to discover the laws of magic and become a god.
+\item \textsc{Hermione Granger} is doing better than him in every class except broomstick riding.
+\item \textsc{Draco Malfoy} is exactly what you would expect an eleven-year-old boy to be like if Darth Vader were his doting father.
+\item \textsc{Professor Quirrell} is living his lifelong dream of teaching Defense Against the Dark Arts, or as he prefers to call his class, Battle Magic. His students are all wondering what's going to go wrong with the Defense Professor this time.
+\item \textsc{Dumbledore} is either insane, or playing some vastly deeper game which involved setting fire to a chicken.
+\item \textsc{Minerva Mcgonagall} needs to go off somewhere private and scream for a while.
 \end{itemize}
 
-\begin{center}
-Presenting:
-
-HARRY POTTER AND THE METHODS OF RATIONALITY
-
-You ain't guessin' where this one's going.
-\end{center}
+% \begin{center}
+% Presenting:
+% 
+% \textsc{Harry Potter and the Methods of Rationality}
+% 
+% You ain't guessin' where this one's going.
+% \end{center}
 
 \section*{Some notes}
-The opinions of characters in this story are not necessarily those of the author. What warm!Harry thinks is \emph{often} meant as a good pattern to follow, especially if Harry thinks about how he can cite scientific studies to back up a particular principle. But not everything Harry does or thinks is a good idea. That wouldn't work as a story. And the less warm characters may sometimes have valuable lessons to offer, but those lessons may also be dangerously double-edged.
+The opinions of characters in this story are not necessarily those of the author. What (warm!) Harry thinks is \emph{often} meant as a good pattern to follow, especially if Harry thinks about how he can cite scientific studies to back up a particular principle. But not everything Harry does or thinks is a good idea. That wouldn't work as a story. And the less warm characters may sometimes have valuable lessons to offer, but those lessons may also be dangerously double-edged.
```

Comment:
"warm!Harry" is deliberate, it's fanfic notation for alternate states/timelines of characters, I think?

Reply:
oki, changed back